docs/xdr/usecases/playbook/whoIs.md

```diff
@@ -0,0 +1,35 @@
+# WhoIs module configuration
+
+This use case describes how to use WhoIs module to retrieve information.
```

Comment:
Suggestion: "in order to enrich an IP address, a domain name or a URL"

(based on the description of the query field in the Whois module)